marklogic-data-hub-api/src/main/java/com/marklogic/hub/impl/HubClientImpl.java

```diff
@@ -58,6 +60,11 @@ public DatabaseClient getStagingClient() {
         return stagingClient;
     }
 
+    @Override
+    public DatabaseClient getStagingClient(String databaseName) {
```

Comment:
Shouldn't we do something similar to allow another database to be the final?
I recall in standup mention of a staging-test and final-test.

Reply:
@aniamy That is a good question. For performance reasons, we want to use an app server that is associated with a content database to avoid invoking across databases. So if the target database is FINAL, we want to use the FINAL app server, etc. We are only adding the ability to specify a database to the STAGING app server-client since functionally speaking there isn't a difference in performance or outcome between using the FINAL or STAGING app server when the target database isn't associated with either.